feat: add roketo integration

PinkiNice opened the pull request:
Add "Create Roketo Stream" button to stream Near and Fungible Tokens via Roketo

ggoshanov committed:
feat: add roketo integration

AngelBlock commented on src/Dao.jsx:
```diff
@@ -757,11 +770,143 @@ const Dao = () => {
+        const isFt = paymentOption === "FT";
+
+        // Its a roketo testnet address, better move it to environments 
+        const roketoContractAddress = 'dev-1635510732093-17387698050424';
```
Can you use the same contracts for consistency?

PinkiNice replied:
Hey didn't understand quite right what do you mean by using the same contracts? 
Could explain that a little bit more?

Mainnet Roketo contact deployed here: `roketodapp.near`

AngelBlock replied:
If you are using (deploying) your own sputinkdao smart contract you better to fork the repo and deploy UI to your own GitHub or other service, as this is a reference / mockup implementation which uses specific smart contract on testnet and mainnet for testing. There is no option (yet) in this tool to select or switch smart contracts (factory) I'm afraid, so if you need to use your own, its your only option for now.

In this case, you don't need to make a PR.

PinkiNice committed:
replace roketo testnet address with mainnet

Replaced testnet roketo address to mainnet.
Saving testnet address here just for case:
Roketo testnet: `dev-1635510732093-17387698050424`

ggoshanov committed:
add Roketo contract configs to env
